examples/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_subdirectory(test-patterns)
 if(TLRENDER_BUILD_GL)
-    add_subdirectory(play-glfw)
+  add_subdirectory(play-glfw)
```

Comment:
It's generally better to make formatting changes separately so it's easier to review the functional changes. But if the tab is removed it should be replaced with 4 spaces. I've accidentally added a bunch of tabs and should probably do a global search and replace.